### PR TITLE
refactor: Match CNI go tests

### PR DESCRIPTION
**Pull request by jpayne3506**
**Reason for Change**:

Did not match the design of other go tests, slowing down development.

**Issue Fixed**:

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [x] relevant PR labels added

**Notes**:

### CODE REVIEW

File: test/integration/datapath/datapath_windows_test.go [hunk 1]
```diff
@@ -123,14 +120,33 @@ func TestDatapathWin(t *testing.T) {
...
+	ctx := context.Background()
+
+	t.Log("Get REST config")
+	restConfig := k8sutils.MustGetRestConfig(t)
 
+	t.Log("Create Clientset")
+	clientset, _ := k8sutils.MustGetClientset()
+
+	setupWindowsEnvironment(t)
+	podLabelSelector := k8sutils.CreateLabelSelector(podLabelKey, podPrefix)
```
Comment (paulyufan2) [inline test/integration/datapath/datapath_windows_test.go:142]: Hi John, can you please rebased your branch, I think these codes are already there

Comment (jpayne3506) [reply test/integration/datapath/datapath_windows_test.go:142]: Just checked master, they are not.

**Review by paulyufan2 — DISMISSED** [review]
lgtm

File: test/integration/datapath/datapath_windows_test.go [hunk 2]
```diff
@@ -123,14 +120,33 @@ func TestDatapathWin(t *testing.T) {
...
+}
+
+func TestDatapathWin(t *testing.T) {
+	ctx := context.Background()
+
+	t.Log("Get REST config")
+	restConfig := k8sutils.MustGetRestConfig(t)
 
+	t.Log("Create Clientset")
+	clientset, _ := k8sutils.MustGetClientset()
```
Comment (vipul-21) [inline test/integration/datapath/datapath_windows_test.go:139]: Why are we ignoring the error ?

File: test/integration/datapath/datapath_windows_test.go [hunk 3]
```diff
@@ -123,14 +120,33 @@ func TestDatapathWin(t *testing.T) {
 
 		pods, err := k8sutils.GetPodsByNode(ctx, clientset, *podNamespace, podLabelSelector, node.Name)
 		if err != nil {
-			require.NoError(t, err)
+			t.Fatal(err)
 		}
 		if len(pods.Items) <= 1 {
-			require.NoError(t, errors.New("Less than 2 pods on node"))
+			t.Fatal("Less than 2 pods on node")
```
Comment (vipul-21) [inline test/integration/datapath/datapath_windows_test.go:126]: Good to have node name  as well ?

File: test/integration/datapath/datapath_windows_test.go (hunk 2 above)
Comment (jpayne3506) [reply test/integration/datapath/datapath_windows_test.go:139]: Good question! Going to change this.

**Review by vipul-21 — APPROVED** [review]

**Pull request merged**